code/tools/make_munki_mpkg.sh

```diff
@@ -341,10 +346,11 @@ done
 # Set permissions.
 chmod -R go-w "$ADMINROOT/usr/local/munki"
 chmod +x "$ADMINROOT/usr/local/munki"
+
 # make paths.d file
 mkdir -p "$ADMINROOT/private/etc/paths.d"
 echo "/usr/local/munki" > "$ADMINROOT/private/etc/paths.d/munki"
-chmod -R 755 "$ADMINROOT/private"
+chmod -R 755 "$ADMINROOT/private/etc/paths.d"
```

Comment:
I guess that's a valid point. While they are the same permissions, it might be assumed that someone could have changed these values.